test/integration/crud/find.test.js

```diff
@@ -2577,4 +2577,42 @@ describe('Find', function () {
       });
     }
   });
+
+  context('when passed an ObjectId instance as the filter', () => {
+    let client;
+    let findsStarted;
+
+    beforeEach(async function () {
```

Comment:
```suggestion
    beforeEach(function () {
```

This doesn't need to be async